src/types/verifiableCredentials/index.ts

```diff
@@ -0,0 +1,51 @@
+export interface Credential {
+  context: string[];
+  id: string;
+  type: string[];
+  issuer: string | Issuer;
+  issuanceDate: Date;
+  expirationDate?: Date;
+  credentialSubject: Subject;
+  credentialStatus?: Status;
+}
+
+export interface VerifiableCredential extends Credential {
+  proof?: Proof[];
+}
+
+export interface Presentation {
+  context: string[];
+  id?: string;
+  type: string[];
+  verifiableCredential?: VerifiableCredential[];
+  holder?: string;
+}
+
+export interface VerifiablePresentation extends Presentation {
+  proof?: Proof[];
+}
+
+export interface Issuer {
+  id?: string;
+}
+
+export interface Subject {
+  id?: string;
+  claims: Record<string, ClaimValue>;
+}
+
+export interface Status {
+  id: string;
+  type: string;
+}
+
+export type ClaimValue = string | ClaimValue[] | { [key: string]: ClaimValue };
+
+export interface Proof {
```

Comment:
I understand that this is a spec related name, but I'd say `Proof` is also a generic name in our context, It might be better changed to `CredentialsProof`. 

Why? Because we already have other kinds of `Proofs` like `MerkleProof` type, `RLNFullProof` type, `SemaphoreProof`, `ZkProof` related types. 
See `@src/types/proof`

Reply:
Sounds good, fixed